apluslms_roman/cli.py

```diff
@@ -379,6 +413,142 @@ def config_print_action(context):
         print("---\n# project config")
         yaml_dump(get_config(context)._data, stdout)
 
+def init_config(context):
+    cwd = getcwd()
+    # don't create if config already exists
+    try:
+        config_path = RomanConfig.find_from(cwd).path
+        # if the folder is legacy, find_from won't error
+        # so we need to check if the file actually exists
+        if not isfile(config_path):
+            raise RomanConfigError
+        exit(1, "{} already exists in {}.\nNo new Roman config created.".format(
+            basename(config_path), cwd)
+        )
+    except RomanConfigError:
+        filename = DEFAULT_NAME + ".yml"
+        config = LEGACY_CONFIG if is_legacy_configuration(cwd) else DEFAULT_CONFIG
+        filepath = path_join(cwd, filename)
+        with open(filepath, 'w') as outfile:
+            yaml_dump(config, outfile)
+        print("{} successfully created in {}".format(filename, cwd))
+
+def set_args(context):
+    if context.args.user == context.args.project:
+        raise Exception("Choose either user settings or project settings")
+    if context.args.user:
+        document = context.settings
+    else:
+        document = context.config
+
+    schema = document.validator.schema
+    for val in context.args.values:
+        try:
+            key, val = val.split('=')
```

Comment:
Using `val.split('=', 1)` would allow `FOOBAR=FOO=FOO` and resuls `key="FOOBAR"; val="FOO=FOO"`.